packages/excalidraw/components/App.tsx

```diff
@@ -3730,7 +3728,7 @@ class App extends React.Component<AppProps, AppState> {
     },
   );
 
-  private onSceneUpdated = () => {
+  private triggerRender = () => {
```

Comment:
`onSceneUpdated()` was a bad name for something that only cares about triggering a render